client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -21,185 +21,208 @@ import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+								<ReduxFormToggle
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="" />
-								<span>
-									{ translate( 'Email address or website URL' ) }
-								</span>
-							</FormLabel>
-
-							<FormLabel>
-								<ReduxFormRadio
+									name="enableRegistration"
+									text={ translate( 'Enable account creation during submission' ) } />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Includes account creation on the listing submission form, to allow ' +
+										'non-registered users to create an account and submit a job listing simultaneously.' ) }
+								</FormSettingExplanation>
+
+								<ReduxFormToggle
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="email" />
-								<span>
-									{ translate( 'Email addresses only' ) }
-								</span>
-							</FormLabel>
-
-							<FormLabel>
-								<ReduxFormRadio
+									name="generateUsername"
+									text="Generate usernames from email addresses" />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+										'email address. If this is not enabled, a "username" field will display instead.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset>
+								<FormLabel>
+									{ translate( 'Role' ) }
+								</FormLabel>
+								<ReduxFormSelect
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="url" />
-								<span>
-									{ translate( 'Website URLs only' ) }
-								</span>
-							</FormLabel>
-						</FormFieldset>
-					</Card>
-				</FormSection>
-			</form>
-		</div>
-	);
-};
-
-JobSubmission.propTypes = {
-	isDisabled: PropTypes.bool,
-	translate: PropTypes.func,
-};
+									name="role">
+									<option value="editor">{ translate( 'Editor' ) }</option>
+									<option value="author">{ translate( 'Author' ) }</option>
+									<option value="contributor">{ translate( 'Contributor' ) }</option>
+									<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+									<option value="teacher">{ translate( 'Teacher' ) }</option>
+									<option value="employer">{ translate( 'Employer' ) }</option>
+									<option value="customer">{ translate( 'Customer' ) }</option>
+									<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+								</ReduxFormSelect>
+								<FormSettingExplanation>
+									{ translate( 'Any new accounts created during submission will have this role. ' +
+										'If you haven\'t enabled account creation during submission in the options above, ' +
+										'your own method of assigning roles will apply.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+						</Card>
+					</FormSection>
+				</form>
+
+				<form>
+					<FormSection name="approval">
+						<SectionHeader label={ translate( 'Approval' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Waiting approval' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpjm-save-job-submission-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).